test/events.backup.js

```diff
@@ -1,3 +1,5 @@
+import '../example/app.js';
```

Comment:
I didn't really fix this file, but I think this is one thing it'd need. The jury is out on whether we fix or rewrite this file.